design/v6-bundle/project/styles_v7.css

```
@@ -0,0 +1,331 @@
+@import url('styles_v3.css');
```

Stylelint (17.10.0)

[error] Expected "url('styles_v3.css')" to be "'styles_v3.css'" (import-notation)

(import-notation)